src/styles/components/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  transition-duration: 0.3s;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    position: relative;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+    position: relative;
+    padding: 23px 0;
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    text-transform: uppercase;
```

Comment:
Font-size should be set in `rem` units instead of `px` for better accessibility and responsiveness. Additionally, `font-weight` should be `bold` or `normal`, as numeric values can vary across different fonts and might not give the expected result. Check the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) for guidance.